### PR TITLE
fix: che #20437: Updating a warning about community devfile registry regression on Eclipse Che hosted by Red Hat

### PR DESCRIPTION
**Pull request by ibuziuk**
## What does this pull request change?
Updating a warning about community devfile registry regression on Eclipse Che hosted by Red Hat

## What issues does this pull request fix or reference?

Related to the regression in the product https://issues.redhat.com/browse/CRW-2116

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [x] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)
- [x] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [x] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

**Comment by ibuziuk** [comment]
@themr0c once merged, please cherry-pick to the branch that corresponds to the status of the up-to-date website. The change should be visible on https://www.eclipse.org/che/docs/che-7/hosted-che/hosted-che/#_differences_between_eclipse_che_and_codeready_workspaces

**Review by themr0c — APPROVED** [review]

**Comment by themr0c** [comment]
Should we commit on a date or leave the date and release undefined?

**Comment by ibuziuk** [comment]
let's not commit to any date, current ETA is mid-October CRW 2.12

File: modules/hosted-che/partials/ref_about-hosted-che.adoc
Comment (themr0c) [inline modules/hosted-che/partials/ref_about-hosted-che.adoc]: @ibuziuk If we don't commit to any date, would following work for you?

```suggestion
The community-supported devfile registry has been temporarily disabled on the link:https://workspaces.openshift.com/[Eclipse Che hosted by Red Hat] due to a defect in the product. This devfile registry will be re-enabled for Hosted Che with an upcoming release.
```

**Comment by themr0c** [comment]
@ericwill we need your review and approval.

**Review by svor — APPROVED** [review]

**Comment by ibuziuk** [comment]
@themr0c sorry, why this trivial PR has not been merged for more than 3 weeks?

**Pull request merged**
